server/util.go

```diff
@@ -279,7 +282,6 @@ func AddCommands(rootCmd *cobra.Command, defaultNodeHome string, appCreator type
 	rootCmd.AddCommand(
 		startCmd,
 		tmcmd.ResetAllCmd,
-		tendermintCmd,
```

Comment:
Was deleting this an accident? Not sure how the tm commands get in without this line

Reply:
No I moved it to the tendermint sub-command with all the other Tendermint proxied commands.